test/e2e/framework/pod/create.go

```diff
@@ -33,6 +33,21 @@ var (
 	BusyBoxImage = imageutils.GetE2EImage(imageutils.BusyBox)
 )
 
+// PodInfo is a struct containing all arguments for creating a pod.
+// SELinux testing requires to pass HostIPC and HostPID as booleansi arguments.
+type PodInfo struct {
+	ns                  string
```

Comment:
I think these fields will have to be public so that they can be used outside of the package

Reply:
Okay. I will change that.